tests/test_aiodns.py

```diff
@@ -21,94 +22,94 @@
 
 
 class DNSTest(unittest.TestCase):
-    def setUp(self):
+    def setUp(self) -> None:
         if sys.platform == 'win32':
             asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
         self.loop = asyncio.new_event_loop()
         self.addCleanup(self.loop.close)
         self.resolver = aiodns.DNSResolver(loop=self.loop, timeout=5.0)
         self.resolver.nameservers = ['8.8.8.8']
 
-    def tearDown(self):
-        self.resolver = None
+    def tearDown(self) -> None:
+        self.resolver = None  # type: ignore[assignment]
 
-    def test_query_a(self):
+    def test_query_a(self) -> None:
         f = self.resolver.query('google.com', 'A')
         result = self.loop.run_until_complete(f)
 
-    def test_query_async_await(self):
-        async def f():
+    def test_query_async_await(self) -> None:
+        async def f() -> Any:
```

Comment:
Same with all the `Any`s in the file. If the above return types are sorted, then these can be replaced here too.